mix.exs

```diff
@@ -72,6 +72,13 @@ defmodule Membrane.RTC.Engine.MixProject do
       {:membrane_mp4_plugin, "~> 0.16.0", optional: true},
       {:membrane_aac_plugin, "~> 0.12.0", optional: true},
       {:membrane_aac_fdk_plugin, "~> 0.13.0", optional: true},
+      {:membrane_opus_plugin, "~> 0.15.0", optional: true},
+
+      # Test deps
+      {:membrane_file_plugin, "~> 0.12.0"},
+      {:membrane_h264_ffmpeg_plugin, "~> 0.21.5"},
+      {:membrane_flv_plugin, "~> 0.2.0", only: :test, runtime: false},
+      {:bimap, "~> 1.2", override: true},
```

Comment:
We can't make `membrane_file_plugin` to test only, because in `membrane_rtp_vp8_plugin` it is used without `only: :test` and the same is with `membrane_h264_ffmpeg_plugin` that is used in `membrane_webrtc_plugin`.

Reply:
Bimap had override because `membrane_flv_plugin` use a different version than another plugin, but as we remove flv this is no longer an issue.

Reply:
That sounds like we should fix membrane_rtp_vp8_plugin :thinking: